assets/js/highscore.js

```diff
@@ -0,0 +1,18 @@
+// Render Highscore from Local Storage
+const highScoreList = function () {
+  const latestScores = JSON.parse(localStorage.getItem("user-input") || "[]");
+
+  for (let i = 0; i < latestScores.length; i++) {
```

Comment:
Would be great to see you implement a .forEach method here instead of the for loop. Functionally nothing is wrong with this, just the iterative array methods in javascript are cleaner to read and easier to write! https://www.w3schools.com/js/js_array_iteration.asp